python/xoscar/context.pyx

```diff
@@ -294,12 +325,31 @@ cdef class ClientActorContext(BaseActorContext):
         context = self._get_backend_context(address)
         return context.buffer_ref(address, buf)
 
-    def copy_to(self, local_buffers: list, remote_buffer_refs: List[BufferRef], block_size: Optional[int] = None):
-        if len(local_buffers) == 0 or len(remote_buffer_refs) == 0:
-            raise ValueError("Nothing to transfer since the length of `local_buffers` or `remote_buffer_refs` is 0.")
-        address = remote_buffer_refs[0].address
+    def file_object_ref(self, str address, object file_object) -> FileObjectRef:
         context = self._get_backend_context(address)
-        return context.copy_to(local_buffers, remote_buffer_refs, block_size)
+        return context.file_object_ref(address, file_object)
+
+    def copy_to(self, local_buffers_or_fileobjs: list, remote_refs: List[Union[BufferRef, FileObjectRef]], block_size: Optional[int] = None):
+        if len(local_buffers_or_fileobjs) == 0 or len(remote_refs) == 0:
+            raise ValueError("Nothing to transfer since the length of `local_buffers_or_fileobjs` or `remote_refs` is 0.")
+        assert (
+            len({ref.address for ref in remote_refs}) == 1
+        ), "remote_refs for `copy_to` can support only 1 destination"
```

Comment:
Why define the parameter `remote_refs` as list but check the len of value == 1?

Reply:
> Why define the parameter `remote_refs` as list but check the len of value == 1?

Here's checking whether there is only on remote address, i.e. the length of ``set`` of remote addresses is 1.

Reply:
I got it. The copy_to API is too complex.